homematic/rootfs/etc/cont-init.d/env.sh

```diff
@@ -11,6 +11,7 @@ mkdir -p /data/hs485d
 mkdir -p /data/userprofiles
 
 ln -s /data/userprofiles /etc/config/userprofiles
+ln -s /data/groups.gson /opt/hm/etc/config/groups.gson
```

Comment:
```
mkdir -p /share/hm-firmware
mkdir -p /share/hmip-firmware
mkdir -p /data/crRFD
mkdir -p /data/rfd
mkdir -p /data/hs485d
mkdir -p /data/userprofiles

# Init files
touch /data/groups.gson
touch /data/hmip_user.conf
touch /data/rega_user.conf
touch /data/homematic.regadom
touch /data/userprofiles/userAckInstallWizard_Admin

# Persist
ln -s /data/userprofiles /etc/config/userprofiles
ln -s /data/groups.gson /opt/hm/etc/config/groups.gson
```

Reply:
Add it to init files and we persist it later. I think that structure is simpler to maintain longtime

Reply:
alright, that is easier to read. I also checked that Homematic is fine with the empty (touched) groups.gson